packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -34,21 +35,44 @@ export class Carequality implements CarequalityAPI {
   static ORG_ENDPOINT = "/Organization";
   readonly api: AxiosInstance;
   readonly apiKey: string;
+  private httpsAgent: Agent;
+  private rsaPrivateKey: string;
+  private passphrase: string;
 
   /**
    * Creates a new instance of the Carequality API client pertaining to an
    * organization to make requests on behalf of.
    *
+   * @param orgCert         The certificate (public key) for the organization.
+   * @param rsaPrivateKey   An RSA key corresponding to the specified orgCert.
+   * @param passphrase      The passphrase to decrypt the private key.
    * @param apiKey          The API key to use for authentication.
    * @param apiMode         Optional, the mode the client will be running. Defaults to staging.
    * @param options         Optional parameters
    * @param options.timeout Connection timeout in milliseconds, default 120 seconds.
    */
-  constructor(
-    apiKey: string,
-    apiMode: APIMode = APIMode.production,
-    options: { timeout?: number } = {}
-  ) {
+  constructor({
+    orgCert,
+    rsaPrivateKey,
+    passphrase,
+    apiKey,
+    apiMode = APIMode.production,
+    options = {},
+  }: {
+    orgCert: string;
+    rsaPrivateKey: string;
+    passphrase: string;
+    apiKey: string;
+    apiMode: APIMode;
+    options?: { timeout?: number };
+  }) {
+    this.rsaPrivateKey = rsaPrivateKey;
+    this.passphrase = passphrase;
+    this.httpsAgent = new Agent({
+      cert: orgCert,
+      key: this.rsaPrivateKey,
+      passphrase: this.passphrase,
```

Comment:
If this is two way TLS (we have been calling this mTLS 😄 ) then you will need one more parameter here called "ca" which contains the root cert to verify the servers certificate. 
The idea of mTLS is that both sides verify each other. If we don't verify them, that might be the source of your production woes. 
So if they are using a private ceret you will want to add one more env var with the appropriate root certificate that verifies the clients certificate. 

if they use a public cert, then you typically do not need to manually specify the CA certificate in your client's configuration. The https module, along with most HTTPS clients, includes a list of trusted CAs (like VeriSign, Let's Encrypt, etc.), and it automatically trusts certificates issued by these authorities.